examples-standalone/aspnetcore-data/Controllers/BlogsController.cs

```diff
@@ -0,0 +1,67 @@
+using System.Linq;
+using Microsoft.AspNetCore.Mvc;
+using Kendo.Mvc.UI;
+using Kendo.Mvc.Extensions;
+using aspnetcore_data.Models;
+
+namespace aspnetcore_data.Controllers
+{
+    public class BlogsController : Controller
+    {
+        private readonly BloggingContext _context;
+
+        public BlogsController(BloggingContext context)
+        {
+
+            _context = context;
+        }
+
+        [Route("api/Blogs")]
+        [HttpGet]
+        public JsonResult GetProducts([DataSourceRequest] DataSourceRequest request)
+        {
+            var result = Json(this._context.Blog.ToDataSourceResult(request));
+            return result;
+        }
+
+        [Route("api/Blogs/create")]
+        [HttpPost]

```

Comment:
Not an important issue, as its not the focus of the demo, but it's considered a good REST API practice to use nouns only (no verbs like create/edit/delete) for route names especially when performing crud operations. The HTTP method is the one that indicates what the action is. It plays the role of the verb so to say. For example:

`GET api/Blogs` => retrieves all resources in the collection (optionally filtered by a query param)
`GET api/Blogs/{id}` => retrieves the specific resource
`POST api/Blogs` => creates a new resource in the targeted collection
`PUT api/Blogs/{id}` => replaces an existing resource
`DELETE api/Blogs/{id}` => deletes an existing resource